blocks/footer/footer.js

```diff
@@ -15,13 +29,72 @@ export default async function decorate(block) {
   if (resp.ok) {
     const html = await resp.text();
 
-    // decorate footer DOM
-    const footer = document.createElement('div');
-    footer.innerHTML = html;
+    // Create container for footer
+    const footerContainer = document.createElement('div');
+    footerContainer.classList.add('footer-container');
+    footerContainer.innerHTML = html;
+
+    const slogan = footerContainer.querySelector('div:nth-child(2) > p:nth-child(1)');
+    slogan.classList.add('footer-slogan');
+
+    footerContainer.querySelectorAll('ul')
+      .forEach((list, index) => {
+        list.classList.add(`footer-menu-list-${index}`);
+      });
+
+    // add horizontal line
+    const list = footerContainer.querySelector('ul');
+    const horizontalLine = document.createElement('hr');
+    list.after(horizontalLine);
+
+    const rightSide = footerContainer.querySelector('div:nth-child(3)');
+    // add target="_blank" to all links in right side
+    rightSide.querySelectorAll('a').forEach((link) => {
+      link.target = '_blank';
+    });
+    rightSide.classList.add('right-side');
+
+    // Create link for powered by fitness logo
+    const poweredByFitnessImg = rightSide.querySelector('p:nth-last-child(2)');
+    const poweredByFitnessImgLink = createImageLink(poweredByFitnessImg, 'https://www.24hourfitness.com/');
+
+    // Create link for try workout logo
+    const tryWorkoutImg = rightSide.querySelector('p:last-child');
+    const tryWorkoutImgLink = createImageLink(tryWorkoutImg, 'https://www.24hourfitness.com/programs/24go/');
+
+    rightSide.append(poweredByFitnessImgLink);
+    rightSide.append(tryWorkoutImgLink);
+
+    const leftSide = footerContainer.querySelector('div:first-child');
+    leftSide.classList.add('left-side');
+
+    // Create link for Brand logo
+    const brandImg = leftSide.querySelector('picture');
+    const brandImgLink = createImageLink(brandImg, '/');
+    leftSide.append(brandImgLink);
+
+    const middleDiv = footerContainer.querySelector('div:nth-child(2)');
+    middleDiv.classList.add('middle-div');
+
+    // Trademark
+    const trademarkContainer = document.createElement('div');
+    trademarkContainer.classList.add('trademark-container');
+    const trademark = middleDiv.querySelector('p:last-child');
+    trademark.classList.add('trade-mark');
+    trademarkContainer.append(trademark);
+    middleDiv.append(trademarkContainer);
 
-    footer.querySelector('p:nth-child(2)').classList.add('footer-slogan');
+    // Top button
+    const topBtnContainer = document.createElement('div');
+    topBtnContainer.classList.add('top-btn-container');
+    const topBtn = document.createElement('a');
+    topBtn.href = '#top';
+    topBtn.className = 'top-btn';
```

Comment:
let's use base this on the franklin default button. 

```suggestion
    const topBtnContainer = document.createElement('div');
    topBtnContainer.classList.add('button-container', 'jump-to-top-container');
    const topBtn = document.createElement('a');
    topBtn.href = '#top';
    topBtn.className = 'button primary jump-to-top';
```